src/Omega_h_refine.cpp

```diff
@@ -76,16 +76,30 @@ static void refine_element_based(Mesh* mesh, AdaptOpts const& opts) {
     transfer_refine(mesh, opts.xfer_opts, &new_mesh, keys2edges, keys2midverts,
         ent_dim, keys2prods, prods2new_ents, same_ents2old_ents,
         same_ents2new_ents);
+
     old_lows2new_lows = old_ents2new_ents;
   }
   *mesh = new_mesh;
 }
 
 static bool refine(Mesh* mesh, AdaptOpts const& opts) {
+
+  mesh->change_all_rcFieldsTorc();
```

Comment:
I assume this does nothing if all the rcFields are already stored in reduced sized tags (i.e., not `Mesh` tags)?

Reply:
No, the structure is such that it is never called if the rcFields are stored in reduced sized tags